resources/ctl.rb

```diff
@@ -103,12 +103,12 @@
 end
 
 action :configure do
-  if shell_out('sensuctl user list').error?
-    converge_by 'Reconfiguring sensuctl' do
-      execute 'configure sensuctl' do
-        command sensuctl_configure_cmd
-        sensitive true unless new_resource.debug
-      end
+  converge_by 'Reconfiguring sensuctl' do
+    config = sensuctl_cluster_config
+    execute 'configure sensuctl' do
+      command sensuctl_configure_cmd
+      sensitive true unless new_resource.debug
+      not_if { config['api-url'] == new_resource.backend_url }
```

Comment:
this still would not allow you to reconfigure the credentials using the same URL, I am wondering if we should perhaps have a configure and reconfigure actions with their own distinct logic as I can't seem to find a good middle ground that gives us what we want for both scenarios.